### PR TITLE
feat: add auth models; shifting authentication to apps folder

### PR DESCRIPTION
**Pull request by akhil7philip**

### CODE REVIEW

File: car_wash/settings.py [hunk 1]
```diff
@@ -23,28 +23,33 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-x^av-oc%u-0$fu)c+m4=#yo-m#wcrb$uj#)+n#746job@=yr&s"
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
```
Comment (Keith3895) [inline car_wash/settings.py:26]: add these keys to the .env_example as well.

File: car_wash/settings.py [hunk 2]
```diff
@@ -23,28 +23,33 @@
...
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv('DEBUG')
 
 ALLOWED_HOSTS = [
-    "10.0.2.2",
-    "localhost"
+    os.getenv('ALLOWED_HOSTS')
```
Comment (Keith3895) [inline car_wash/settings.py:32]: do we get array vals In env? In node, it usually comes as a string.

**Review by Keith3895 — CHANGES_REQUESTED** [review]

File: car_wash/settings.py (hunk 2 above)
Comment (akhil7philip) [reply car_wash/settings.py:32]: My bad; let me add the correction

File: car_wash/settings.py (hunk 1 above)
Comment (akhil7philip) [reply car_wash/settings.py:26]: Will push the changes

**Pull request closed without merging**